src/components/AstronautsPage/components/AstronautsList/components/AstronautsItem/components/AstronautsItemImage/AstronautsItemImage.tsx

```diff
@@ -1,13 +1,17 @@
 import React from 'react';
 import { CardMedia } from '@mui/material';
-import { astronautsData } from '../../../../../../mocks/astronautsData';
+// import { astronautsData } from '../../../../../../mocks/astronautsData';
 import './AstronautsItemImage.scss';
 
-export const AstronautsItemImage = () => (
+interface AstronautsItemImage {
+  image: string;
+}
+
+export const AstronautsItemImage = ({ image }: AstronautsItemImage) => (
```

Comment:
probably we can use there React.memo wrapping

Reply:
you are right, added